build.ps1

```diff
@@ -13,14 +13,6 @@ Param(
     [string[]]$ScriptArgs
 )
 
-# delete cake directory if script changed
-$CakeDirPath = Join-Path $PSScriptRoot "cake"
```

Comment:
Oops; this is used on line 17. Is that line still necessary, or should the rest of the script be updated?